src/SdfEntityCreator.cc

```diff
@@ -339,23 +340,23 @@ Entity SdfEntityCreator::CreateEntities(const sdf::Model *_model)
   auto ent = this->CreateEntities(_model, false);
 
   // Load all model plugins afterwards, so we get scoped name for nested models.
-  for (const auto &[entity, element] : this->dataPtr->newModels)
+  for (const auto &[entity, plugins] : this->dataPtr->newModels)
   {
-    this->dataPtr->eventManager->Emit<events::LoadPlugins>(entity, element);
+    this->dataPtr->eventManager->Emit<events::LoadSdfPlugins>(entity, plugins);
```

Comment:
Users that are listening to the old event would be broken by this change, right? Should we emit both events on this branch?

Reply:
Done in [afd30fb](https://github.com/gazebosim/gz-sim/pull/1352/commits/afd30fbb79c7c81cf9d7bb056179fa6cd92a0d94)